tools/executors/translate/translate.ts

```diff
@@ -8,14 +8,66 @@ const translate = new v2.Translate({
   keyFilename: './tools/executors/translate/files/google-cloud-secret.json',
 });
 
-export async function translateText(text: string[], target: string) {
+export async function translateText(
+  text: string,
+  language: string
+): Promise<string> {
   // Translates the text into the target language. "text" can be a string for
   // translating a single piece of text, or an array of strings for translating
   // multiple texts.
   // Un-comment below once you have google-cloud-secret.json file ready
   // let [translations] = await translate.translate(text, target);
 
   // Comment below once you have google-cloud-secret.json file ready
-  const translations = text.map((t) => 'Translated ' + t);
-  return translations;
+  const translation = language + '-' + text;
+  return translation;
+}
+
+export async function translateJSON(
+  jsonData: { data: any[] },
+  language: string
+): Promise<{ data: any[] }> {
+  let updatedJSON: { data: any[] } = { data: [] };
+
+  const data = jsonData.data;
+
+  const updatedData = await translateDataArray(data, language);
```

Comment:
Why do we need all this code? you have the `data` array property from the JSON file, and we should translate its elements

Reply:
It's needed for nested jsons.

Reply:
For example?

Reply:
I don't think we have such a case. Our data is always a list of strings

Reply:
@shhdharmen now I see the food example, you are right 😀

Reply:
Chexk if you support this please

Reply:
Checked for food, worked fine.